skrub/tests/test_sentence_encoder.py

```diff
@@ -0,0 +1,139 @@
+import pandas as pd
+import pytest
+from numpy.testing import assert_array_equal
+from sklearn.base import clone
+
+from skrub import SentenceEncoder
+from skrub._on_each_column import RejectColumn
+from skrub._sentence_encoder import ModelNotFound
+
+pytest.importorskip("sentence_transformers")
+
+
+@pytest.fixture
+def encoder():
+    """Create a fixture such that we set the common parameters of the encoder.
+
+    Here, we have two constraints:
+
+    - We want a really small model to not blow up the memory of the GitHub Actions
```

Comment:
if it is still a problem with this model we can also mock the sentence transformer with something that has the same interface but returns random values

Reply:
With the instabilities in torch, transformers, and sentence-transformers, maybe it's best not to trust them fully and run the full tests anyway?